src/documents/tests/test_file_handling.py

```diff
@@ -290,88 +293,6 @@ def test_asn(self):
         self.assertEqual(generate_filename(d1), "652 - the_doc.pdf")
         self.assertEqual(generate_filename(d2), "none - the_doc.pdf")
 
-    @override_settings(FILENAME_FORMAT="{tags[type]}")
```

Comment:
Im trying to understand, are these still supported or is this what changed in 1012_fix_archive_files.py ?

Reply:
`tags` was not a documented variable, so I've removed it entirely rather than attempt to convert it.  A template can access tags already in several ways now, so if anyone happens to be using it, I think there's a better "official way" now